api/controllers/SessionsController.js

```diff
@@ -5,52 +5,39 @@
  * @help        :: See https://sailsjs.com/docs/concepts/actions
  */
 
-const fs = require('fs');
-const path = require('path');
 const conditionHelper = require('../utils/ConditionHelpers');
 
 module.exports = {
-  index: function (req, res) {
-
-    let directory = path.resolve(__dirname, '../../sessions');
+  index: async function (req, res) {
+
+    const reports = await MedicalReport.findAll({limit: 10, attributes: ['id', 'firstName', 'lastName', 'socialInsuranceNumber']});
+    sails.log(`Query returned: ${JSON.stringify(reports)}`);
+    const viewModel = reports.map(x =>  {
+      return {
+        id: x.id,
+        dateRecieved: 'Not Implemented',
+        applicant: `${x.lastName}, ${x.firstName}`,
```

Comment:
you can get this right from the model as `x.fullName`

Reply:
x.fullName is has the names in the wrong order